llvm/lib/Target/ARM/ARMInstrThumb.td

```diff
@@ -1768,7 +1768,7 @@ let isBranch = 1, isTerminator = 1, isBarrier = 1, isIndirectBranch = 1 in {
 
 // In Thumb1, "nop" is encoded as a "mov r8, r8". Technically, the bf00
 // encoding is available on ARMv6K, but we don't differentiate that finely.
```

Comment:
Should the comment be updated?

Reply:
No, I don't think so. The `mov r8, r8` is still used for Thumb1. I added other code to make sure its not used in cases with the bf00 encoding available.

Reply:
We also still dont differentiate for ARMv6k